samza-core/src/main/java/org/apache/samza/zk/ScheduleAfterDebounceTime.java

```diff
@@ -144,27 +157,27 @@ private Runnable getScheduleableAction(String actionName, Runnable runnable) {
         } else {
           LOG.debug("Action: {} completed successfully.", actionName);
         }
-      } catch (Throwable t) {
-        LOG.error("Execution of action: {} failed.", actionName, t);
-        doCleanUpOnTaskException(t);
+      } catch (Exception exception) {
```

Comment:
I don't see these changes in hot-fix branch.
Can you provide context on why are we changing this to `exception` from `throwable`?

Reply:
Reverted this change.